test/DynamicEater.test.ts

```
@@ -41,40 +42,40 @@
 	}
 
 	describe("Merge with dynamic rates", () => {
-		it("Should get price as percent at start of bonding curve", async () => {
+		it.only("Should get price as percent at start of bonding curve", async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)